# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1364755528 14400
#      Sun Mar 31 14:45:28 2013 -0400
# Node ID 2a294ee8efc0fd6409a8cc2f9b88d07312c79583
# Parent  424463a801349f551b9be42633b55bc8810dc3ed
* NEWS: include strjoin() in list of new functions for Octave 3.8.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -138,18 +138,18 @@ Summary of important user-visible change
     are now available (provided that Octave is compiled with support for
     Java enabled):
 
       helpdlg    listdlg   questdlg
       inputdlg   msgbox    warndlg
 
  ** Other new functions added in 3.8.0:
 
-      betaincinv   dawson     fminsearch  polyeig     tetramesh
-      cmpermute    erfcinv    importdata  rgbplot
+      betaincinv   dawson     fminsearch  polyeig     strjoin
+      cmpermute    erfcinv    importdata  rgbplot     tetramesh
       cmunique     erfi       iscolormap  shrinkfaces
       colorcube    findfigs   lines       splinefit
 
  ** Deprecated functions.
 
     The following functions were deprecated in Octave 3.4 and have been
     removed from Octave 3.8.
                                            
